src/types.h

```diff
@@ -104,9 +104,9 @@ typedef Kokkos::View<const T_V_FLOAT*>    t_mass_const; // Mass
 
 //Cabana
 using t_tuple = Cabana::MemberTypes<T_FLOAT[3], T_FLOAT[3], T_FLOAT[3],
-                                    T_INT, T_INT, T_FLOAT>;
+                                    T_INT, T_INT, T_FLOAT, T_FLOAT>;
 enum TypeNames { Positions = 0, Velocities = 1, Forces = 2,
-                 Types = 3, IDs = 4, Charges = 5 };
+                 Types = 3, IDs = 4, Charges = 5, Potentials = 6 };
```

Comment:
It would be nice to avoid adding another particle prop - LJ avoids this by using a `Kokkos::parallel_reduce` for energies instead of keeping them per particle (and a separate function call).

As you have it now you would just need to reduce across `p` and return it in `compute_energy` for it to work with the existing `PotE::compute`